nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessPropagation.java

```diff
@@ -923,9 +923,9 @@ public TransferResult<Nullness, NullnessStore> visitMethodInvocation(
         node, callee, node.getArguments(), values(input), thenUpdates, bothUpdates);
     NullnessHint nullnessHint =
         handler.onDataflowVisitMethodInvocation(
-            node, state, apContext, values(input), thenUpdates, elseUpdates, bothUpdates);
+            node, callee, state, apContext, values(input), thenUpdates, elseUpdates, bothUpdates);
     Nullness nullness = returnValueNullness(node, input, nullnessHint);
-    if (booleanReturnType(node)) {
+    if (booleanReturnType(callee)) {
```

Comment:
note that `getSymbol(MethodInvocationTree tree)` never returns null (maybe this was different in previous errorprone versions?)